src/repl/scala/tools/nsc/interpreter/Imports.scala

```diff
@@ -153,14 +153,13 @@ trait Imports {
       trailingBraces append "}\n"+ request.postwrap +"\n"
       accessPath append s".$iw"
     }
-    def addWrapper() {
+    def addWrapper(): Unit =
```

Comment:
I shall resist checking the sordid history. There is a kind of long tail to these idioms.